src/sass/_config.scss

```diff
@@ -126,26 +126,31 @@ $toplayerZIndex: 1000;
 // Breakpoints
 $breakpointMedium: 768px !default;
 $breakpointLarge: 1024px !default;
+$breakpointXLarge: 1280px !default;
 
 $breakpointsMap: (
   'small-only': '(max-width: #{$breakpointMedium - 1px})',
   'medium-only': '(min-width: #{$breakpointMedium}) and (max-width: #{$breakpointLarge - 1px})',
   'medium-down': '(max-width: #{$breakpointLarge - 1px})',
   'medium-up': '(min-width: #{$breakpointMedium})',
-  'large-only': '(min-width: #{$breakpointLarge})'
+  'large-only': '(min-width: #{$breakpointLarge})',
```

Comment:
since we are adding xl now, you don't have to add max-width here, like for medium-only?

Reply:
that's true

Reply:
@Malinicz This is breaking change and we can do that in separate PR. A lot of places uses `large-only` in the meaning of `large-up`

Reply:
ok I see, sure let's do that separately 👍

Reply:
Second thing, for `sg-hide` class, we need `sg-hide-for-large-DOWN` because it works in opposite way. It should hide content until wide of the screen is XL.

Reply:
@bartoszlorek makes sense

Reply:
@bartoszlorek added `large-down` to generate `sg-hide-for-large-down` class